src/renderers/dom/shared/ReactBrowserEventEmitter.js

```diff
@@ -132,40 +139,44 @@ var ReactBrowserEventEmitter = Object.assign({}, ReactEventEmitterMixin, {
    * @param {string} registrationName Name of listener (e.g. `onClick`).
    * @param {object} contentDocumentHandle Document which owns the container
    */
-  listenTo: function(registrationName, contentDocumentHandle) {
+  listenTo: function(registrationName, contentDocumentHandle, node) {
     var mountAt = contentDocumentHandle;
-    var isListening = getListeningForDocument(mountAt);
+    var isDocListening = getListeningForDocument(mountAt);
     var dependencies =
       EventPluginRegistry.registrationNameDependencies[registrationName];
 
     for (var i = 0; i < dependencies.length; i++) {
       var dependency = dependencies[i];
-      if (
-        !(isListening.hasOwnProperty(dependency) && isListening[dependency])
+
+      if (localOnly.hasOwnProperty(dependency)) {
+        ReactDOMEventListener.trapBubbledEvent(
```

Comment:
SimulateNative actually wouldn't cover it because that hooks in right at the top level listener of the event system, whereas I'm claiming that that listener would be called twice. Can you point me to where you're saying the old listener gets removed?